ldb/models.py

```diff
@@ -331,6 +335,19 @@ def age(self):
         else:
             return today.year - born.year
 
+    def clean(self):
+        if self.email_forward:
+            if self.ldap_username is None:
+                raise ValidationError(
```

Comment:
Repace with `self.add_error(None, "CH LDAP username should be set to enable forwarding.")` or `self.add_error("email_forward", "...")`

https://docs.djangoproject.com/en/3.0/ref/forms/api/#django.forms.Form.add_error

Reply:
Hmm: AttributeError, 'Person' object has no attribute 'add_error'

Reply:
We can do a `raise ValidationError` for a field though: https://docs.djangoproject.com/en/2.2/ref/models/instances/#django.db.models.Model.clean

Reply:
Ah, that's for a form. 🙈